core/templates/dev/head/profile/profile.html

```diff
@@ -71,9 +71,8 @@ <h2 class="oppia-profile-username-small-screen">
       <p class="oppia-profile-subject-interest-container">
         Interests:
         <span ng-if="subjectInterests.length > 0"><br>
-          <span ng-repeat="interest in subjectInterests track by $index">
-            <span class="oppia-profile-subject-interest"><[interest]></span>
-          </span>
+        <!-- The following tag should remain on a single line due to http://bugzilla.mozilla.org/show_bug.cgi?id=922773. -->
+          <span ng-repeat="interest in subjectInterests track by $index"><span class="oppia-profile-subject-interest"><[interest]></span></span>
```

Comment:
Yes, we need to collapse he outer span.  Firefox doesn't collapse the line feed between the lines, so I had to get rid of them in the code.